make.bat

```diff
@@ -59,8 +59,10 @@ for /f "usebackq tokens=*" %%i in (`"%VsWhereDir%\Microsoft Visual Studio\Instal
 	set InstallDir=%%i
 )
 
-if exist "%InstallDir%\Common7\Tools\vsdevcmd.bat" (
-	call "%InstallDir%\Common7\Tools\vsdevcmd.bat" -arch=%HostArch% -host_arch=%HostArch% -no_logo
+if exist %InstallDir%\Common7\Tools\vsdevcmd.bat (
+	call %InstallDir%\Common7\Tools\vsdevcmd.bat -arch=%HostArch% -host_arch=%HostArch% -no_logo
+) else if exist "%VsWhereDir%/Microsoft Visual Studio 14.0/Common7/Tools/vsdevcmd.bat" (
```

Comment:
I am not sure BAT files support else if.
https://stackoverflow.com/questions/11081735/how-to-use-if-else-structure-in-a-batch-file

Reply:
Yes, it support.